regtests/ww3_tic1.1/input_IC5_M1/ww3_grid_1000m.inp

```diff
@@ -7,6 +7,11 @@ $
    F T F F F T
   60. 60. 60. 60.
 $
+$ IC5VEMOD defines which sea ice model is selected
+$ IC5VEMOD = 1 - Extended Fox and Squire model (EFS)
+$ IC5VEMOD = 2 - Robinson and Palmer model (RP)
+$ IC5VEMOD = 3 - Order 3 Power Law model of Meylan et al. (2018; M2)
+&SIC5 IC5VEMOD = 1 /
```

Comment:
@QingxiangLiu Did you add tests for IC5VEMOD=2 and 3? or is there a reason for not adding them?

Reply:
Hi @aliabdolali,  I added tests for *IC5VEMOD = 2* and *IC5VEMOD = 3* as well in the subfolders **input_IC5_M2** and **input_IC5_M3**.